README.md

```diff
@@ -11,160 +8,142 @@
+![Quick runthrough of basic IPFS Companion features](https://gateway.ipfs.io/ipfs/QmShhuV4SXTfqBxerxxPvmvbu8Z98LzePtVMmq47AU7PTL)
+
+
+
+| <img src="https://unpkg.com/@browser-logos/firefox/firefox_16x16.png" width="16" height="16"> [Firefox](https://www.mozilla.org/firefox/new/) \| [Firefox for Android](https://play.google.com/store/apps/details?id=org.mozilla.firefox) | <img src="https://unpkg.com/@browser-logos/chrome/chrome_16x16.png" width="16" height="16"> [Chrome](https://www.google.com/chrome/) \| <img src="https://unpkg.com/@browser-logos/brave/brave_16x16.png" width="16" height="16"> [Brave](https://brave.com/) \| <img src="https://unpkg.com/@browser-logos/opera/opera_16x16.png" width="16" height="16"> [Opera](https://www.opera.com/)  \| <img src="https://unpkg.com/@browser-logos/edge/edge_16x16.png" width="16" height="16"> [Edge](https://www.microsoftedgeinsider.com/)
 |------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | [![Install From AMO](https://ipfs.io/ipfs/QmWNa64XjA78QvK3zG2593bSMizkDXXcubDHjnRDYUivqt)<br>![](https://img.shields.io/amo/users/ipfs-companion?label=AMO%20users&style=social)](https://addons.mozilla.org/firefox/addon/ipfs-companion/) | [![Install from Chrome Store](https://ipfs.io/ipfs/QmU4Qm5YEKy5yHmdAgU2fD7PjZLgrYTUUbxTydqG2QK3TT)<br>![](https://img.shields.io/chrome-web-store/users/nibjojkomfdiaoajekhjakgkdhaomnch?label=Chrome%20Web%20Store%20users&style=social)](https://chrome.google.com/webstore/detail/ipfs-companion/nibjojkomfdiaoajekhjakgkdhaomnch) |
 
-## Lead Maintainer
+## Lead maintainer
 
-[Marcin Rataj](https://github.com/lidel)
+[Marcin Rataj](https://github.com/lidel), with help from the [IPFS GUI working group](https://github.com/ipfs/ipfs-gui).
 
-## Table of Contents
+## Table of contents
 
-- [Background](#background)
+- [About IPFS Companion](#about-ipfs-companion)
 - [Features](#features)
-- [Install](#install)
+- [Install](#install-ipfs-companion)
 - [Contribute](#contribute)
-- [Troubleshooting](#troubleshooting)
-- [Privacy Policy](#privacy-policy)
-- [License](#license)
+- [Help & Troubleshooting](#help--troubleshooting)
+- [Privacy & Licenses](#privacy--license)
 
-## Background
+## About IPFS Companion
 
-This add-on enables everyone to access IPFS resources the way they were meant: from locally running IPFS node :-)
+IPFS Companion harnesses the power of your locally running IPFS node (either through the IPFS Desktop app or the command-line daemon) directly inside your favorite Chromium-based or Firefox browser, enabling support for `ipfs://` addresses, automatic IPFS gateway loading of websites and file paths, easy IPFS file import and sharing, and more.
```

Comment:
Some users may learn that Desktop exists from this very paragraph. 
We should add link – would github one be enough?

```suggestion
IPFS Companion harnesses the power of your locally running IPFS node (either through the [IPFS Desktop](https://github.com/ipfs-shipyard/ipfs-desktop/#ipfs-desktop) app or the command-line daemon) directly inside your favorite Chromium-based or Firefox browser, enabling support for `ipfs://` addresses, automatic IPFS gateway loading of websites and file paths, easy IPFS file import and sharing, and more.
```

Reply:
It's the best we've got right now 😊 sounds good.